plan/logical_plan_builder.go

```diff
@@ -992,6 +997,17 @@ func (b *planBuilder) resolveGbyExprs(p LogicalPlan, gby *ast.GroupByClause, fie
 		exprs = append(exprs, expr)
 		p = np
 	}
+	if b.ctx.GetSessionVars().SQLMode.HasOnlyFullGroupBy() {
+		for index, field := range fields {
+			if _, ok := field.Expr.(*ast.AggregateFuncExpr); ok {
+				continue
+			}
+			if !field.InGroupBy {
+				b.err = ErrFieldNotInGroupBy.GenByArgs(index+1, field.Text())
```

Comment:
Could you add a test case for this?

Reply:
the test case is in executor/aggregate_test.go:TestOnlyFullGroupBy

Reply:
consider non-aggregated columns that functionally dependent on (uniquely determined by) `GROUP BY` columns ?

Reply:
yes, Reject queries for which the select list refer to nonaggregated columns that are neither named in the GROUP BY clause nor are functionally dependent on (uniquely determined by) GROUP BY columns.

Reply:
sorry, I'll fix the bug